common.js

```diff
@@ -138,6 +138,13 @@ var vcwg = {
       authors: ['Orie Steele', 'Michael Jones'],
       status: 'ED',
       publisher: 'W3C Verifiable Credentials Working Group'
+    },
+    'VC-SPECS-DIR': {
+      title: 'Verifiable Credentials Specifications Directory',
+      href: 'https://w3c.github.io/vc-specs-dir/',
+      authors: ['Manu Sporny'],
+      status: 'ED',
```

Comment:
Does this have to become a NOTE / TR in order for us to reference it informatively?

Reply:
Resolving since this comment is not a request to change this PR (and the question has been answered).